tds/src/main/webapp/WEB-INF/classes/log4j2.xml

```diff
@@ -170,6 +170,12 @@
       <appender-ref ref="threddsServlet"/>
     </logger>
 
+    <!-- ASW SDK -->
+    <!-- Note: Do not log at "debug" level it in your production environments because it could log sensitive data! -->
```

Comment:
Extraneous "it".

And I'd probably go with "sensitive authentication data" to make clear that the issue isn't the data being served, it is aws authentication information.

Reply:
oh, well...

Reply:
oops haha, I will fix it :)